packages/app-project/stores/Project.js

```diff
@@ -5,6 +5,7 @@ import numberString from './types/numberString'
 
 const Project = types
   .model('Project', {
+    about_pages: types.frozen([]),
```

Comment:
The `Project.spec.js` file tests for the model properties, so let's add an existential check for this new one.